templates/data/generic_component_data.yaml

```diff
@@ -1019,30 +1019,6 @@ templates:
                     global:
                       - "jobConfig_postsubmit"
                       - "disable_testgrid"
-      - to: ../prow/jobs/kyma/components/permission-controller/permission-controller-generic.yaml
-        jobConfigs:
-          - repoName: "github.com/kyma-project/kyma"
-            jobs:
-              - jobConfig:
-                  path: components/permission-controller
-                  args:
-                    - "/home/prow/go/src/github.com/kyma-project/kyma/components/permission-controller"
-                  run_if_changed: "^components/permission-controller/|^common/makefiles/"
-                  release_since: "1.10"
```

Comment:
Instead of removing whole template, which will remove jobs for all releases, you can add
`release_until: "1.24"` to generate jobs for all supported releases until 1.24 inclusive (so 1.23 and 1.24)